### PR TITLE
Add redis configuration to celery documentation

### LINKED ISSUE #3147: Can't start celery beat & workers
After adding the celery configuration according to the documentation, trying to start either the celery beat or worker results in the following error, even after linking the redis container to all lemur containers.

```
Traceback (most recent call last):
  File "/usr/local/src/lemur/venv/lib/python3.6/site-packages/celery/app/utils.py", line 373, in find_app
    sym = symbol_by_name(app, imp=imp)
  File "/usr/local/src/lemur/venv/lib/python3.6/site-packages/celery/bin/base.py", line 534, in symbol_by_name
    return imports.symbol_by_name(name, imp=imp)
  File "/usr/local/src/lemur/venv/lib/python3.6/site-packages/kombu/utils/imports.py", line 62, in symbol_by_name
    return getattr(module, cls_name) if cls_name else module
AttributeError: module 'lemur.common' has no attribute 'celery'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/src/lemur/venv/lib/python3.6/site-packages/redis/connection.py", line 559, in connect
    sock = self._connect()
  File "/usr/local/src/lemur/venv/lib/python3.6/site-packages/redis/connection.py", line 615, in _connect
    raise err
  File "/usr/local/src/lemur/venv/lib/python3.6/site-packages/redis/connection.py", line 603, in _connect
    sock.connect(socket_address)
OSError: [Errno 99] Cannot assign requested address

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/src/lemur/lemur/common/redis.py", line 31, in redis
    red.set("test", 0)
  File "/usr/local/src/lemur/venv/lib/python3.6/site-packages/redis/client.py", line 1801, in set
    return self.execute_command('SET', *pieces)
  File "/usr/local/src/lemur/venv/lib/python3.6/site-packages/redis/client.py", line 898, in execute_command
    conn = self.connection or pool.get_connection(command_name, **options)
  File "/usr/local/src/lemur/venv/lib/python3.6/site-packages/redis/connection.py", line 1192, in get_connection
    connection.connect()
  File "/usr/local/src/lemur/venv/lib/python3.6/site-packages/redis/connection.py", line 563, in connect
    raise ConnectionError(self._error_message(e))
redis.exceptions.ConnectionError: Error 99 connecting to localhost:6379. Cannot assign requested address.
```

It seems like somehow lemur still tries to connect to `localhost:6379`, even though a different redis host ist configured as `CELERY_RESULT_BACKEND` as well as `CELERY_BROKER_URL`

### PR DESCRIPTION
**Pull request by peschmae**
Currently when adding the celery config, the RedisHandler still tries to connect to `locahost:6379` even if a different `CELERY_BROKER_URL ` and `CELERY_BACKEND` is configured.

This documentation update, adds the configurations necessary, for the RedisHandler to properly use the same redis instance as celery.

Fixes #3147

### CODE REVIEW

File: docs/production/index.rst [hunk 1]
```diff
@@ -390,6 +390,9 @@ Here are the Celery configuration variables that should be set::
     CELERY_IMPORTS = ('lemur.common.celery')
     CELERY_TIMEZONE = 'UTC'
 
+    REDIS_HOST="your_redis_url"
+    REDIS_PORT="6379"
+
```
Comment (hosseinsh) [inline docs/production/index.rst:395]: what about adding this:
```
REDIS_DB=0   

Out of the box, every Redis instance supports 16 databases. The default database (`REDIS_DB`) is  set to 0, however, you can use any of the databases from 0-15. Via `redis.conf` more databases can be supported. 
```

File: docs/production/index.rst [hunk 2]
```diff
@@ -390,6 +390,9 @@ Here are the Celery configuration variables that should be set::
     CELERY_IMPORTS = ('lemur.common.celery')
     CELERY_TIMEZONE = 'UTC'
 
+    REDIS_HOST="your_redis_url"
+    REDIS_PORT="6379"
```
Comment (hosseinsh) [inline docs/production/index.rst:394]: `REDIS_PORT=6379`

**Review by hosseinsh — COMMENTED** [review]
Thank you Matthias for the PR. I have added some suggestion to improve the documentation.

**Comment by peschmae** [comment]
I've updated the pull request, with your suggestions, and added an example how to add the database to the `redis://` URL as well.

**Comment by hosseinsh** [comment]
Thanks Matthias for the detailed documentation.

**Pull request merged**